build.gradle

```diff
@@ -150,21 +150,30 @@ subprojects {
             transitive = false
         }
     }
+}
 
-    release {
-        // Disable check snapshots temporarily
-        buildTasks = []
-        failOnSnapshotDependencies = false
-        failOnCommitNeeded = false
-        versionPropertyFile = 'gradle.properties'
-        tagTemplate = 'v${version}'
-        git {
-            // To release from any branch
-            requireBranch = null
-        }
+def moduleVersion = project.version
+if (moduleVersion.indexOf('-') != -1) {
+    moduleVersion = moduleVersion.substring(0, moduleVersion.indexOf('-'))
+}
+
+release {
+    failOnPublishNeeded = false
+
+    buildTasks = ['build']
+    versionPropertyFile = 'gradle.properties'
+    tagTemplate = 'v$version'
+
+    git {
+        requireBranch = "release-${moduleVersion}"
+        pushToRemote = 'origin'
     }
 }
 
+task build {
+    dependsOn('sql-ballerina:build')
+}
+
```

Comment:
We can remove this by specifying `buildTasks = []`. Basically, line 163 should be like this `buildTasks = []`